service/src/test/java/bio/terra/workspace/pact/LzApiTest.java

```diff
@@ -0,0 +1,612 @@
+        .uponReceiving("A request to list landing zone defs")
+        .method("GET")
+        .path("/api/landingzones/definitions/v1/azure")
+        .willRespondWith()
+        .body(definitionsResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "lzs")
+  public RequestResponsePact listAzureLandingZoneResources(PactDslWithProvider builder) {
+    var resourcesResponseShape =
+        new PactDslJsonBody().uuid("id").eachLike("resources", resourcesByTypeShape);
+
+    return builder
+        .given("An existing landing zone")
+        .uponReceiving("A request to list the landing zone resources")
+        .method("GET")
+        .pathFromProviderState(
+            "/api/landingzones/v1/azure/${landingZoneId}/resources",
+            "/api/landingzones/v1/azure/%s/resources".formatted(landingZoneId))
+        .willRespondWith()
+        .body(resourcesResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "lzs")
+  public RequestResponsePact getResourceQuota(PactDslWithProvider builder) {
+    var quotaResponseShape =
+        new PactDslJsonBody()
+            .uuid("landingZoneId")
+            .stringType("azureResourceId")
+            .stringType("resourceType")
+            .object("quotaValues");
+
+    return builder
+        .given("An existing landing zone")
+        .uponReceiving("A request to get a resource's quota")
+        .method("GET")
+        .pathFromProviderState(
+            "/api/landingzones/v1/azure/${landingZoneId}/resource-quota",
+            "/api/landingzones/v1/azure/%s/resource-quota".formatted(landingZoneId))
+        .query(String.format("azureResourceId=%s", azureResourceId))
+        .willRespondWith()
+        .body(quotaResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Test
+  @PactTestFor(pactMethod = "startCreateLandingZone", pactVersion = PactSpecVersion.V3)
+  void startCreateLandingZone() throws InterruptedException {
+    var result =
+        landingZoneService.startLandingZoneCreationJob(
+            new BearerToken("fake"),
+            "jobId",
+            UUID.randomUUID(),
+            "LZDefinition",
+            "v1",
+            List.of(new ApiAzureLandingZoneParameter().key("key").value("value")),
+            UUID.randomUUID(),
+            "https://example.com/async");
+
+    assertNotNull(result);
+    assertNotNull(result.getJobReport());
+    assertThat(true, equalTo(false));
+  }
+
+  @Test
+  @PactTestFor(pactMethod = "createLandingZone_notAuthorized", pactVersion = PactSpecVersion.V3)
+  void startCreateLandignZone_notAuthorized() {
+    assertThrows(
+        LandingZoneServiceAuthorizationException.class,
+        () ->
+            landingZoneService.startLandingZoneCreationJob(
+                new BearerToken("fake"),
+                "jobId",
+                UUID.randomUUID(),
```

Comment:
We should avoid using random UUIDs here.

The value will end up in the generated contract, which will change on every run, and cause the broker to re-run validation every time it's pushed, since it looks like the contract changed.